upload.prx.org/pipeline.yml

```diff
@@ -2,7 +2,7 @@ AWSTemplateFormatVersion: "2010-09-09"
 Description: >
   Creates a CodePipeline that is used to handle CI/CD for the upload lambda
 Parameters:
-  OpsDebugMessagesSNSTopicArn:
+  OpsWarnMessagesSNSTopicArn:
```

Comment:
I think that this was included originally because I copied over from another pipeline template that did have it -- so, happy to take it out.